internal/cli/atlas/clusters/onlinearchive/create.go

```diff
@@ -109,9 +105,6 @@ To learn more about online archives, see https://www.mongodb.com/docs/atlas/onli
   # Create an online archive for the sample_mflix.movies collection in a cluster named myTestCluster using a profile named egAtlasProfile when the current date is greater than the value of the released date plus 2 days. Data is partitioned based on the title field, year field, and released field from the documents in the collection:
   %[1]s clusters onlineArchive create --clusterName myTestCluster --db sample_mflix --collection movies --dateField released --archiveAfter 2 --partition title,year --output json -P egAtlasProfile `, cli.ExampleAtlasEntryPoint()),
 		PreRunE: func(cmd *cobra.Command, args []string) error {
-			if len(opts.partitions) > maxPartitions {
```

Comment:
removing validation given APIs are validating it and there are some conditions on the validation